src/gdpr/ConsentManagementProvider.js

```diff
@@ -6,7 +6,7 @@ import { default as installCMPStub } from '@iabtcf/stub';
 
 import { debug, getCookieDomain, getJSON } from '../shared/utils';
 
-export const CMP_VERSION = 5; // Increment to force modal again
+export const CMP_VERSION = 6; // Increment to force modal again
```

Comment:
```suggestion
export const CMP_VERSION = 7; // Increment to force modal again
```
`6` has already been used by experiment released in PL for Experian (see #306)